backend/app/services/subscription/execution.py

```
@@ -740,8 +743,118 @@ def update_execution_status(
+            if loop and loop.is_running():
+                # Schedule as a task if we're already in an async context
+                asyncio.create_task(
+                    subscription_notification_dispatcher.dispatch_execution_notifications(
+                        db,
+                        subscription_id=execution.subscription_id,
+                        execution_id=execution.id,
+                        subscription_display_name=subscription_display_name,
+                        result_summary=result_summary,
+                        status=status,
+                    )
+                )
```

Ruff (0.14.14)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)